README.md

```diff
@@ -40,7 +40,7 @@ import 'package:supabase_flutter/supabase_flutter.dart';
 void main() {
```

Comment:
```suggestion
void main() async {
```

Reply:
@bdlukaa good catch, thanks. What about omitting `void` here for brevity as it is unnecessary? (Tho I'm not familiar with the conventions the repo owners may want to follow.)

Reply:
keep the `void` type. It's useful to make sure nothing is returned to `main`